databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -626,8 +626,10 @@ impl proto::val_server::Val for broker::DataBroker {
     //                   e.g. if sending an unsupported enum value
     //              - if the published value is out of the min/max range specified
     //
-    //    - Provider returns BatchActuateStreamResponse <- Databroker sends BatchActuateStreamRequest
-    //        No error definition, a BatchActuateStreamResponse is expected from provider.
+    //    - Databroker sends BatchActuateStreamRequest -> Provider may return one or more BatchActuateStreamResponse upon error,
+    //        but should for performance reasons send nothing upon success.
+    //        It is up to the provider to decide if the stream shall be closed,
+    //        as of today Databroker will not react on the received error message.
```

Comment:
I see it as a reasonable behavior if you as a provider get a lot of junk from the Databroker, like if you get a lot of `BatchActuateStreamRequest` where the signal id does not match the IDs provided in the `ProvideActuationRequest` you have sent, or the values does not match the value limitations the Provider have internally. I think it is up to the Provider to decide how many errors it will tolerate before it gets pissed off and closes the stream.

Like if a provider has received 1000 requests and all of them have been faulty, why should keep the stream open